frontend/awx/administration/execution-environments/hooks/useExecutionEnvironmentsFilters.tsx

```diff
@@ -1,17 +1,24 @@
 import { useDynamicToolbarFilters } from '../../../common/useDynamicFilters';
+import {
+  useCreatedByToolbarFilter,
+  useModifiedByToolbarFilter,
+} from '../../../common/awx-toolbar-filters';
 
 export function useExecutionEnvironmentsFilters() {
```

Comment:
This should take an additional prop, `url` so it can filter down the list of filterable values when it's within a tabbed view. Currently the UI shows all EEs within an Org:

<img width="1625" alt="filtered_ee" src="https://github.com/ansible/ansible-ui/assets/2293210/17043887-b70d-4039-a014-1d1d86145957">

I would recommend using a similar approach to this [PR](https://github.com/ansible/ansible-ui/pull/1960/files). Let me know if you have any questions - happy to discuss further.